src/cli/cli_commissioner.cpp

```diff
@@ -88,6 +88,46 @@ template <> otError Commissioner::Process<Cmd("joiner")>(Arg aArgs[])
     const otExtAddress *addrPtr = nullptr;
     otJoinerDiscerner   discerner;
 
+    if (aArgs[0] == "list")
```

Comment:
How about using `"table"` instead of `"list"` for the command here?

For other CLI commands, like `neighbor`, `child` or `router` table we have both `table` and `list` sub-commands. 

- `table` outputs a nicely formatted table.
- while `list` is simpler text format (list of key/value pairs) - `list` is often used by CI tests since easier to parse.

Reply:
Perfect, done.